File: app/src/main/java/com/example/letmebeyourchef/Listeners/DettagliRicettaListener.java
package com.example.letmebeyourchef.Listeners;

import com.example.letmebeyourchef.Models.ResponseFromApiDettagliRicetta;

public interface DettagliRicettaListener {
    void didFetch(ResponseFromApiDettagliRicetta response, String message);
    void didError(String message);
}

File: app/src/main/java/com/example/letmebeyourchef/Listeners/DettagliRicettaListener.kt
package com.example.letmebeyourchef.Listeners

import com.example.letmebeyourchef.Models.ResponseFromApiDettagliRicetta

open interface DettagliRicettaListener {
    fun didFetch(response: ResponseFromApiDettagliRicetta?, message: String?)
    fun didError(message: String?)
}